js/components/thermoscope.js

```diff
@@ -45,12 +47,38 @@ export default class Thermoscope extends PureComponent {
     }
   }
 
+  onMeterChange(value) {
+    let range = MAX_TEMP - MIN_TEMP;
+    let scaledValue = Math.round(value * range);
+    scaledValue = MIN_TEMP != 0 ? scaledValue + MIN_TEMP : scaledValue;
+    this.setState({temperature: scaledValue});
```

Comment:
It seems a little odd that the meter knows about the min and the max, but it has to be re-scaled again when it calls the onChange function. i.e. it uses inputs in scaled units, but outputs unscaled unites. Would it be easier for it to just call the onChange with the scaled value from the start?